### DISCUSSION TITLE
Set default customer address

### DISCUSSION THREAD

**Discussion by tarikrital** [swellstores/community#520] [Help] [answered]
Which API is responsible for setting default address? (`swell-node`)

![image](https://github.com/swellstores/community/assets/52907282/0806a23e-df30-4f69-b337-044fb2b55156)

**Comment by logeshswell** [comment, marked as answer]
`account.shipping.account_address_id` denotes the default address.
https://developers.swell.is/backend-api/accounts/the-account-model

**Reply by tarikrital** [reply]
Thank you, can you show a code of how to modify it?

**Reply by tarikrital** [reply]
I found it using `$set`.

Thank you.
